controllers/rhmi/bootstrapReconciler.go

```diff
@@ -202,14 +202,6 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	}
 	metrics.SetQuota(installation.Status.Quota, installation.Status.ToQuota)
 
-	// temp code for RHOAM, remove once all clusters are upgraded to 1.14
-	// Remove all prometheus rules under redhat/sandbox-rhoam/rhoami-operator
-	phase, err = r.removePrometheusRules(ctx, serverClient, installation.Spec.NamespacePrefix)
```

Comment:
Should the relevant function also be removed now that it's unused ? 🤔 
- https://github.com/integr8ly/integreatly-operator/blob/master/controllers/rhmi/bootstrapReconciler.go#L228

Reply:
Yes it should have been :) I have cleaned up the test around the this function and removed any used functions because of it. Also did a clean up of most of the error golangci-lint would have shown.